app/posts/ecf-v2/2024-07-15-workshop-process.md

```diff
@@ -0,0 +1,69 @@
+* analysing the lead provider API error log
+
+#### Chose the top 4 high level problems that each user group faced, our user groups for the service are the following:
+
+* Lead Providers 
+* Schools
+* Appropriate bodies
+* DFE internal users
+
+#### Held workshops per user group and voted the problems against:
+
+*	the problem being a root problem/symptom of a bigger problem
+*	the problems impact on users and severity of said impact
+*	the problems feasibility
+*	the problems sequencing risk
+*	if solving the problem would promote flexibility
+*	the likelihood of fixing the problem using ECF related digital services alone 
+*	if solving the problem promotes flexibility
+
+Once the voting was complete, we were able to see which of the problems we should treat as a higher priority based on the scores. The team then ideated possible solutions for these problems at the end of each workshop. This was the starting point for the ideation exercise.
+
+### How we approached the ideation exercise
+
+*	Gathered all the solutions the team came up with for ideation
+*	Kept them colour coded to show which user group and problem the solution was in response to 
+*	Grouped them loosely into large themes/ categories e.g. Data/Comms 
+*	Further filtered the groups into more specific solutions e.g. Improving and fixing the data model/ Researching comms with users and how we communicate changes
+*	Conducted a scope exercise with our PM to determine which of the solutions were: **in scope for our services MVP**, **out of scope for MVP** and **completely out of scope for our service**
+*	Began the process with the wider team and discussed if everyone agreed with where the solutions were placed in terms of scope and if they had any other solutions to add
+*	Finally voted on the solutions we deemed in scope against their feasibility, impact they’d have on all our users, and how far they met the services core purpose
+
+We were able to end the workshop with a clear idea of our high level solutions allowing us to move forward with road mapping for our MVP.
+
+### What were the results of the workshop?
+
+Based on our analysis of problems & needs, we considered 26 ideas and narrowed it down to 10 for MVP:
+
+*	improve and fix the data model
+*	improve how schools accounts are created and the rules around them
+*	Show the right data over the Lead provider API (e.g. induction start date)
+*	Have the right rules and validation in the digital service to represent the policy and meet the needs of users
+*	Renaming services and key terms for users or parts of the service
+*	Improve the API documentation
+*	Create improved ways for schools and lead providers to make changes to ECT and mentor records, rather than support or developers
+*	Make the journey of registering ECTs and mentors with DFE more intuitive for schools and help them understand their wider responsibilities
+*	 Join and improve finance and admin tooling for internal users
+*	Create timelines for participants 
```

Comment:
```suggestion
Based on our analysis of problems & needs, we considered 26 ideas and narrowed it down to 10 for MVP:

*	improve and fix the data model
*	improve how schools accounts are created and the rules around them
*	show the right data over the Lead provider API (e.g. a teacher's induction start date)
*	have the right rules and validation in the digital service to represent the policy and meet the needs of users
*	rename and restructure services and key terms for users or parts of the service
*	improve the API documentation
*	create improved ways for schools and lead providers to make changes to ECT and mentor records, rather than support or developers
*	make the journey of registering ECTs and mentors with DFE more intuitive for schools and help them understand their wider responsibilities
*	 join and improve finance and admin tooling for internal users
*	create timelines for participants 
```